repository/Seaside-WebComponents-Core.package/WAWebComponentActionContinuation.class/instance/continue.st

```diff
@@ -0,0 +1,4 @@
+processing
+continue
+	"do not capture state"
```

Comment:
Side-note: since we do not capture state, do we need the contents of the `states` instvar in this class?

Reply:
I see what you meant. The question comes down to how much we want to tie `WAFragmentRenderPhaseContinuation` to `WAFragmentRenderPhaseContinuation`. Right now the `states` inst vars are still filled with `WASnapshot` instances due to two reasons:

- `WAFragmentRenderPhaseContinuation` inherits`WARenderLoopContinuation >> #createActionContinuationWithContext:` and send `#snapshot:renderContext:` to the action phase continuation class
- `WAWebComponentActionContinuation >> #handle:` still sends ends up in `WASessionContinuation >> #handle:`

Reply:
It was not a blocking comment but maybe something to look at further down the road.